tools/FileTestDefs.sh

```diff
@@ -622,7 +622,8 @@ standard_checks+=(enable_if)
 # Check for noexcept
 noexcept() {
     whitelist "$1" \
-              'src/Parallel/StaticSpscQueue.hpp' && \
+              'src/Parallel/StaticSpscQueue.hpp' \
+              "src/Parallel/NodeLock..pp$" && \
```

Comment:
for my education, what does the `$` at the end of the string do?

Reply:
That's end-of-line in regex